src/Lumberjack.js

```diff
@@ -15,24 +15,25 @@
 const { Console } = require('console');
 const { format } = require('util');
 
+// eslint-disable-next-line max-params
```

Comment:
please add a reason why this eslint rule is being disabled

Reply:
I've updated to remove the rule and use a single parameter in 7f36955